src/fidesops/ops/service/privacy_request/request_runner_service.py

```diff
@@ -343,20 +355,58 @@ async def run_privacy_request(
         )
         if not proceed:
             return
-
+        if config.notifications.send_request_completion_notification:
+            try:
+                initiate_privacy_request_completion_email(session, is_access_request, access_result_urls, identity_data)
+            except IdentityNotFoundException as e:
+                logger.error(e)
+                privacy_request.error_processing(db=session)
+                # If dev mode, log traceback
+                await fideslog_graph_failure(
+                    failed_graph_analytics_event(privacy_request, e)
+                )
+                _log_exception(e, config.dev_mode)
+                return
         privacy_request.finished_processing_at = datetime.utcnow()
         AuditLog.create(
             db=session,
             data={
                 "user_id": "system",
                 "privacy_request_id": privacy_request.id,
-                "action": AuditLogAction.finished,
+                "action": AuditLogAction.finished,  # fixme- this was existing code but "finished" isn't an AuditLogAction?
                 "message": "",
             },
         )
         privacy_request.status = PrivacyRequestStatus.complete
-        privacy_request.save(db=session)
         logging.info("Privacy request %s run completed.", privacy_request.id)
+        privacy_request.save(db=session)
+
+
+def initiate_privacy_request_completion_email(session: Session, is_access_request: bool, access_result_urls: List[str], identity_data: Dict[str, Any]) -> None:
+    if is_access_request:
+        if not identity_data.get(ProvidedIdentityType.email.value):
+            # fatal bc user will not get results
+            raise IdentityNotFoundException("Identity email was not found, so access request email could not be sent.")
+        dispatch_email(
+            db=session,
+            action_type=EmailActionType.PRIVACY_REQUEST_COMPLETE_ACCESS,
+            to_email=identity_data.get(ProvidedIdentityType.email.value),
+            email_body_params=AccessRequestCompleteBodyParams(
+                download_links=access_result_urls
+            ),
+        )
+    else:
+        if not identity_data.get(ProvidedIdentityType.email.value):
+            # not fatal bc data was deleted as requested
```

Comment:
I don't consider this fatal, so do not consider this an errored privacy request, but open to handling this as we do access requests here https://github.com/ethyca/fidesops/pull/1282/files#diff-3d5f0b1a3f488cadb9d63aea80c3cae154b472a3a6f066d5963b76c841d1756cR388

Reply:
I've chatted with Sean about this also, and he wishes to consider this an errored request, since regulatory practices require that the subject be notified upon completion of requests.